features/topology-map/plugins/org.opennms.features.topology.plugins.topo.bsm/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -15,9 +15,11 @@
         <argument ref="transactionAwareBeanProxyFactory" />
         <property name="businessServiceManager" ref="businessServiceManager" />
     </bean>
-    <bean id="bsmTopologyHopProvider"
-        class="org.opennms.features.topology.api.support.VertexHopGraphProvider">
+    <bean id="bsmTopologyHopProvider" class="org.opennms.features.topology.api.support.VertexHopGraphProvider">
         <argument ref="bsmTopologyProvider" />
+
+        <property name="name" value="Business Service Monitoring Topology Provider"/>
+        <property name="description" value="This Topology Provider displays the hierarchy of the defined Business Services and their computed operational state."/>
```

Comment:
This Topology Provider displays the hierarchy of the defined Business Services and their computed operational states.